parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -312,6 +312,10 @@ ensureContainerd() {
 ExecStartPost=/sbin/iptables -P FORWARD ACCEPT
 EOF
 
+  if [ "${ARTIFACT_STREAMING_ENABLED}" == "true" ]; then
+    ensureArtifactStreaming || $ERR_ARTIFACT_STREAMING_DOWNLOAD_INSTALL
```

Comment:
```suggestion
    logs_to_events "AKS.CSE.ensureContainerd.enableArtifactStreaming" ensureArtifactStreaming || $ERR_ARTIFACT_STREAMING_DOWNLOAD_INSTALL
```

Reply:
Added it. Yes, it would be good to see the CSE execution time to install this both for regressions and as a data point to decide if all of this should move to the VHD for GA.